src/components/AddBook.js

```diff
@@ -14,6 +14,7 @@ export default function AddBook() {
       category: form.book_category.value,
       author: '',
     };
+    e.target.reset();
     dispatch(addBook(newBook));
   };
   return (
```

Comment:
- [ ] Let us try to match with the design provided. Kindly, compare your implementation with the design provided to meet all the requirements. Please use the same font size, color, and font family.

Check the header section and the add book button

Original design

![db8png](https://user-images.githubusercontent.com/55635977/151552383-00579afc-3d95-4146-a46b-3c30074260ca.png)

Yours 
![db5](https://user-images.githubusercontent.com/55635977/151552361-181a8d0c-aaa8-4299-a42c-0bdfe106a055.png)

Original design

![db9](https://user-images.githubusercontent.com/55635977/151552387-af03bbe4-986b-4e6b-9b2c-5be75312cf75.png)

Your design

![db6](https://user-images.githubusercontent.com/55635977/151552380-243eb5eb-b637-48e1-b8cb-d0ad71cf2dd3.png)

Reply:
Hello @Emmyn5600 ,

Thanks a lot for your insightful review.

On to make the requested changes right away. 🏃🏽